### PR TITLE
Japanese translation (domain.md)

### PR DESCRIPTION
**Pull request by wahapo**
To translate `domain.md` to japanese.

And fix image path.
🙅 https://docs.screwdriver.cd/about/appendix/assets/runtime-model.png
🙆 https://docs.screwdriver.cd/about/assets/runtime-model.png

### CODE REVIEW

File: docs/ja/about/appendix/domain.md [hunk 1]
```diff
@@ -58,9 +60,9 @@ A step is a named action that needs to be performed, usually a single shell comm
 
 ジョブは[ソースコード](#%E3%82%BD%E3%83%BC%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%89)への変更や[ワークフロー](#workflow)からのトリガーで自動的に開始します。UIから手動で開始することも可能です。
 
-#### Pull Requests
+#### プルリクエスト
 
-Pull requests are run separately from existing pipeline jobs. They will only execute steps from the `main` job in the Screwdriver configuration.
+プルリクエストは既存のパイプラインジョブとは別に実行されます。ジョブ名が`main`のものからステップが実行されます。
```
Comment (yokawara) [inline docs/ja/about/appendix/domain.md]: [imo] 後半は「`main`ジョブの（で定義されている）ステップのみ実行されます。」という感じになるかなと思います。
ここは原文の内容も古いので別途修正した方が良さそうですね...

File: docs/ja/about/appendix/domain.md [hunk 2]
```diff
@@ -119,9 +121,11 @@ $ meta get example
 {"coverage":99.95}
 ```
 
+See the [metadata page](../../user-guide/metadata) for more information.
```
Comment (yokawara) [inline docs/ja/about/appendix/domain.md]: in japanese

Comment (yokawara) [reply docs/ja/about/appendix/domain.md]: ここもお願いします
https://github.com/screwdriver-cd/guide/pull/274/files#diff-862b5e20086c021ef998eed31a579645R133

Comment (wahapo) [reply docs/ja/about/appendix/domain.md]: I totally forgot.

File: docs/ja/about/appendix/domain.md (hunk 1 above)
Comment (wahapo) [reply docs/ja/about/appendix/domain.md]: The content of the English document is old, so it needs to be modified separately. For now, I will keep Japanese sentences consistent with English documents.

**Comment by yokawara** [comment]
🙏 https://github.com/screwdriver-cd/guide/pull/274#discussion_r258831702

**Comment by wahapo** [comment]
> https://github.com/screwdriver-cd/guide/pull/274#issuecomment-466245348

Oh, I understood. I did not know exactly what you were talking about.

**Review by yokawara — APPROVED** [review]
👍

**Review by kumada626 — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**